dvc/ignore.py

```diff
@@ -0,0 +1,114 @@
+from dulwich.ignore import match_pattern, read_ignore_patterns
+from dvc.utils.compat import cast_bytes
+
+import os
```

Comment:
standard imports should be higher than others :) Pylint would catch that, need to push a little and finally enable it :slightly_smiling_face: